api/src/main/java/jakarta/data/repository/Select.java

```diff
@@ -51,6 +51,10 @@
  *
  * <p>This annotation must not be used in other locations.</p>
  *
+ * <p>Sort criteria for a repository method that returns a projection
+ * can include any of the returned entity attributes that are sortable.
```

Comment:
Perhaps something like:

```suggestion
 * can include any of the returned entity attributes that are of sortable type.
```